tests/Satooshi/Bundle/CoverallsV1Bundle/Config/ConfiguratorTest.php

```diff
@@ -60,11 +59,10 @@ public function shouldLoadNonExistingYml()
         $this->assertConfiguration($config, $this->srcDir, array($this->cloverXmlPath), $this->jsonPath);
```

Comment:
Oh, but $this->srcDir is not used in assertConfiguration any more. I'll remove this.